tests/testData/maps/LuaFunctions.lua

```diff
@@ -20,6 +20,10 @@ function getAllowedChanges()
 	return {general=true, addonsAll=false, addonsSome=true, swapping=false, playerState = not isSinglePlayer, ownNation = true, ownColor=true, ownTeam=false, aiNation = false, aiColor=false, aiTeam=false}
 end
 
+function isMapPreviewEnabled()
```

Comment:
Any reason to add this here?

Reply:
I added it because other settings are there too.